tests/integ/scala/test_dataframe_suite.py

```diff
@@ -542,71 +542,124 @@ def test_df_stat_approx_quantile(session):
 
 def test_df_stat_crosstab(session):
     cross_tab = (
-        TestData.monthly_sales(session).stat.crosstab("empid", "month").collect()
+        TestData.monthly_sales(session)
+        .stat.crosstab("empid", "month")
+        .sort(col("empid"))
```

Comment:
If we added `sort`, can we still keep the original test on dict? Why do we need to test each field explicitly?

Reply:
this was the original version of test I forked from. I can re-add dictionary based tests but they are slightly harder to debug once we encounter failures. For example, it didn't show that we had `KeyError` once server side changes were released that changed the column names for PIVOT